001-core.php

```diff
@@ -27,3 +27,8 @@ function wpcom_vip_disable_core_update_cap( $caps, $cap ) {
 	}
 	return $caps;
 }
+
+/**
+ * Remove Try Gutenberg callout introduced as part of 4.9.8
+ */
+remove_action( 'try_gutenberg_panel', 'wp_try_gutenberg_panel' );
```

Comment:
Should we move this to the Ramp loader file?

Reply:
This is core related, specific to VIP Go and core compatibility. I think it's better not to mix Ramp in when possible to avoid confusion.